app/views/pets/show.html.erb

```diff
@@ -6,9 +6,9 @@
 <p>Adoption Status: <%= @pet.adoption_status %></p>
 
 <% if favorites.contents.include?(@pet.id) %>
-  <p> This pet is already favorited </p>
+  <%= button_to "Remove from Favorites", "/favorites/#{@pet.id}", method: :delete %>
```

Comment:
I think this would be an awesome opportunity to write an enabled and disabled boolean into this logic.